### PR TITLE
Release bug fixes

### PR DESCRIPTION
**Pull request by javydreamercsw**
This is to fix the issues mentioned by @LegacyKing. Please evaluate and most of all test prior to merge.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -322,7 +324,7 @@ task all(dependsOn: ['build', 'slowtest', 'javadoc', 'buildNsis', 'allReports'])
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.5'
+    gradleVersion = '4.0'
```
Comment (grimreaper) [inline build.gradle]: This requires additional testing. Its likely okay but actually doesn't do anything on its own without regenerating the wrapper and upgrading gradle itself.

Comment (javydreamercsw) [inline build.gradle]: I regenerated the wrapper in a recent commit.

**Comment by LegacyKing** [comment]
The errors in travis aren't pretty...

**Comment by LegacyKing** [comment]
`Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0
	at build_cers5p836boda5mq2dx84xcje$_run_closure25.doCall(/home/travis/build/PCGen/pcgen/build.gradle:272)
The setTestClassesDir(File) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use the setTestClassesDirs(FileCollection) method instead.
	at build_cers5p836boda5mq2dx84xcje$_run_closure25.doCall(/home/travis/build/PCGen/pcgen/build.gradle:272)
:copyMasterSheets
:compileJavaNote: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
:processResources
:classes
:makeplugindirs
:jar-gmgen-plugins FAILED
:autoLintGradle
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':jar-gmgen-plugins'.
> /home/travis/build/PCGen/pcgen/build/classes/main does not exist.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED in 52s
6 actionable tasks: 6 executed
The command "eval ./gradlew assemble " failed. Retrying, 2 of 3.
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0
	at build_cers5p836boda5mq2dx84xcje$_run_closure25.doCall(/home/travis/build/PCGen/pcgen/build.gradle:272)
The setTestClassesDir(File) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use the setTestClassesDirs(FileCollection) method instead.
	at build_cers5p836boda5mq2dx84xcje$_run_closure25.doCall(/home/travis/build/PCGen/pcgen/build.gradle:272)
:copyMasterSheets
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:makeplugindirs
:jar-gmgen-plugins FAILED
:autoLintGradle
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':jar-gmgen-plugins'.
> /home/travis/build/PCGen/pcgen/build/classes/main does not exist.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED in 3s
6 actionable tasks: 4 executed, 2 up-to-date
The command "eval ./gradlew assemble " failed. Retrying, 3 of 3.
Gradle now uses separate output directories for each JVM language, but this build assumes a single directory for all classes from a source set. This behaviour has been deprecated and is scheduled to be removed in Gradle 5.0
	at build_cers5p836boda5mq2dx84xcje$_run_closure25.doCall(/home/travis/build/PCGen/pcgen/build.gradle:272)
The setTestClassesDir(File) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use the setTestClassesDirs(FileCollection) method instead.
	at build_cers5p836boda5mq2dx84xcje$_run_closure25.doCall(/home/travis/build/PCGen/pcgen/build.gradle:272)
:copyMasterSheets
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:makeplugindirs
:jar-gmgen-plugins FAILED
:autoLintGradle
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':jar-gmgen-plugins'.
> /home/travis/build/PCGen/pcgen/build/classes/main does not exist.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
BUILD FAILED in 3s
6 actionable tasks: 4 executed, 2 up-to-date
The command "eval ./gradlew assemble " failed 3 times.
The command "./gradlew assemble" failed and exited with 1 during .
Your build has been stopped.`

File: build.gradle (hunk 1 above)
Comment (grimreaper) [inline build.gradle]: lets revert that for now and make this change separate. Its easier to figure out what broke things with smaller more concrete changes,

**Comment by javydreamercsw** [comment]
Just pushed fixes for those issues in the build. Small details, you can see on the change. I'll be now going thru the release process as documented to see if everything is working.

**Comment by javydreamercsw** [comment]
- [X] Compilation/tests worked
- [X] Installer worked
- [X] Full zip worked

Looks good to me. Again, would appreciate more people giving it a try.

Here are some corrections that need to be made to the release instructions:
- Step 5 can remove clean from the tasks since prepareRelease calls clean as dependency.
- Step 8 not needed. Those are build as part of prepareRelease.

**Pull request merged**